.github/scripts/check_app_name.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+TITLE_FILE="fastlane/metadata/android/en-US/title.txt"
+CANONICAL_TITLE=$(cat "$TITLE_FILE" | tr -d '\r\n')
```

Comment:
The Fastlane title contains a bit of marketing ("Catima - Loyalty Card Wallet"), so it doesn't ever match.

Let's just hardcode this one :)

```suggestion
CANONICAL_TITLE="Catima"
```